apps/api/src/app/invites/entities/invite.entity.ts

```diff
@@ -1,19 +1,18 @@
 /* istanbul ignore file */
-import { ObjectId } from "mongodb"
 import {
     Column,
     Entity,
     Index,
     JoinColumn,
-    ObjectIdColumn,
-    OneToOne
+    OneToOne,
```

Comment:
I found the reason why the invite code is only generated once. `@OnetoOne` can have relation with only one invite code per group.

I found `@ManytoOne` to be the solution here.

Please tell me if I'm wrong!

Reply:
I think OneToOne is correct for invites, but maybe there should by a OneToMany `invites` field in the group entity.

Reply:
Let me check! But you're right I guess

Reply:
Good catch! I'm going to fix it